### DISCUSSION TITLE
Release v1.5.0 / 5.60.0

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2363] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build brings ARM64 support to sandboxie it supports windows 10 and 11 on ARM64, for non ARM users this build brings only a couple of fixes and minor improvements.
The ARM64 build is to be considered pre-release it may still contain bugs, while the regular Intel/AMD builds are regular bug fix releases where no issues are to be expected.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

# Changelog

### Added
- Added support for Windows on ARM64 [#1321](https://github.com/sandboxie-plus/Sandboxie/issues/1321) [#645](https://github.com/sandboxie-plus/Sandboxie/issues/645)
-- Ported SbieDrv for ARM64 
-- Ported low level injection mechanism for ARM64/ARM64EC
-- Ported syscall hooks for ARM64/ARM64EC
-- Ported SbieDll.dll to ARM64/ARM64EC
-- Note: ARM32 on ARM64 is not implemented and will terminate with message SBIE2338
-- Note: When Sandboxie is running, it disables the use of CHPE binaries for x86 processes globally, this is required for the forced processes functionality. This behaviour can be disabled by adding the global option "DisableCHPE=n" to the Sandboxie.ini, then x86 processes started outside the sandbox, instead of being forced, will be terminated with message SBIE2338

### fixed
- fixed issue with Win32 hooks in x86 applications
- Avoid window overlap when editing templates [#2339](https://github.com/sandboxie-plus/Sandboxie/pull/2339) (thanks okrc)
- Fixed the wrong write of OpenWinClass setting UI [#2347](https://github.com/sandboxie-plus/Sandboxie/pull/2347) (thanks okrc)
- Fixed issue about Local Template [#2338](https://github.com/sandboxie-plus/Sandboxie/pull/2338) (thanks okrc)
- fixed Edge WebView compatybility issue [#2350](https://github.com/sandboxie-plus/Sandboxie/issues/2350)
- added provisional workaround for firefox 106 content process sandbox issue
- Fixed bug with Rename Sandbox [2358](https://github.com/sandboxie-plus/Sandboxie/pull/2358) (thanks okrc)

### Changed
- reworked API compatibility check
- Break out process is now available for all users

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.5.0'>Release v1.5.0 / 5.60.0</a>.</em>

**Comment by BilleBarrett** [comment]
Still getting this error

|Time|         |Message|                                                                                                                          

07:59:15.583   chrome.exe (13632): SBIE2203 Failed to communicate with Sandboxie Service:  *GUIPROXY_00000001; MsgId: 9 - chrome.exe [C0000034]   
Again I uninstalled Sandboxie-Plus 1.4.2 and delete everything and then reinstalled 1.50
After reinstall I start checking and testing 1.5.0
BTW Microsoft Edge came up with any errors
I tracked the problem to Security Options [Security hardening]
If I check "Enable all security enhancements" that is when everything stops working 
If I check " Drop rights from Administrators and Power users" things stop working 
As of right now I'm good 
Bill

**Comment by BilleBarrett** [comment]
This was in the Sandboxie Message windows this morning nothing running overnight except Outlook 
& has done it several more times 
WerFault.exe (7140): SBIE2203 Failed to communicate with Sandboxie Service:  *GUIPROXY_00000001; MsgId: 14 - WerFault.exe [FF000000]
